src/markdown.cpp

```diff
@@ -135,6 +142,11 @@ class Trace
       m_resultSet = true;
       m_resultValue = b ? "true" : "false";
     }
+    void setResult(ExplicitPage ep)
```

Comment:
inconsistent code formatting: In other source files, class methods are separated by a single empty line each.